data/human/free worlds 4 epilogue.txt

```diff
@@ -32,7 +32,7 @@ mission "FW Epilogue: Ijs and Katya"
 					goto excitement
 				`	"It's kind of hard to adjust to civilian life."`
 					goto adjust
-				`	"I want to explore the rest of the galaxy, but apparently that part of the game hasn't been implemented yet."`
+				`	"I want to explore the rest of the galaxy, but apparently parts of it are not yet implemented in the current version of the game."`
```

Comment:
```suggestion
				`	"I want to explore the entire rest of the galaxy, but apparently some parts of the game have not been implemented yet."`
```

If the primary concern is that some may leap to the conclusion that there is nothing else at all out there, you can still emphasize that not all of it is waiting without rearranging or drifting from the original sentiment, which is that the game overall still needs expansion (and help doing that), not just the galaxy itself.

This distinction is important for two reasons:

First and foremost, expansion of the galaxy is not the primary focus of improving the game overall, and currently existing stuff will still need improvement once the galaxy is filled.

Secondly, the galaxy -will- become filled, at some point (in theory, even if not in practice), meaning this changed line would then be inappropriate and need changing again once no more galactic additions are wanted.